gulpfile.babel.js

```diff
@@ -3,6 +3,7 @@ import {join} from 'path'
 import {gzipSync} from 'zlib'
 
 // Packages
+// eslint-disable-next-line no-unused-vars
 import _ from 'babel-polyfill'
```

Comment:
When I removed the error the build crashes [(https://travis-ci.org/zeit/styled-jsx/builds/234130358)](https://travis-ci.org/zeit/styled-jsx/builds/234130358).